packages/common/src/utils/index.ts

```diff
@@ -17,3 +18,49 @@ export function getFilePaths(dir: string): string[] | [] {
     } catch (err) {}  // eslint-disable-line no-empty
```

Comment:
I know this PR did not introduce this function, but it's probably a good time to look at it. Ignoring errors is usually a code-smell. In what cases would ignoring exceptions reading file paths be ok?